gen_ziektenplagenexotengroen.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+# Generates the ziektenplagenexotengroen mapfile.
+# TODO rewrite using mappyfile, generalize.
+
+from contextlib import contextmanager
+
+indent: int = 0
+
+
+def p(*strs: list[str]):
+    print("  " * indent, end="")
+    print(strs[0], end="")
+    if len(strs) > 1:
+        print(" ", end="")
+    print_quoted(strs[1:])
+
+
+def q(*strs: list[str]):
+    print("  " * indent, end="")
+    print_quoted(strs)
+
+
+def print_quoted(strs: list[str]):
+    print(*map(repr, strs))
+
+
+@contextmanager
+def block(typ: str):
+    p(typ)
+    global indent
+    indent += 1
+    yield
+    indent -= 1
+    p("END")
+
+
+layers = [
+    ("Eikenprocessierups aanwezig (Laag)", "caterpillar_blue"),
+    ("Eikenprocessierups deels bestreden", "tree_orange"),
+    ("Niet in beheergebied Gemeente Amsterdam", "flag_black"),
+    ("Eikenprocessierups aanwezig (Urgent)", "caterpillar_red"),
+    ("Gemeld", "speechbubble"),
+    ("Eikenprocessierups bestreden", "tree_green"),
+    ("Geen Eikenprocessierups aanwezig", "tree_black"),
+    ("Niet bereikbaar voor bestrijding", "flag_red"),
+    ("Eikenprocessierups aanwezig (Standaard)", "caterpillar_orange"),
+]
+
+print("# GENERATED FILE, DO NOT EDIT.\n\n")
+print("# TEAM: Bor / Beeldschoon\n")
+
+with block("MAP"):
+    p("NAME", "ziektenplagenexotengroen")
+    p("INCLUDE", "header.inc")
+    p("DEBUG", 5)
```

Comment:
Does this need to stay here for the final version?

Reply:
There's lots of different `DEBUG` settings in this repo already. I intend to clean all of them up at a later point.